sdk/servicebus/Microsoft.Azure.ServiceBus/src/Primitives/ConcurrentExpiringSet.cs

```diff
@@ -40,28 +52,38 @@ void ScheduleCleanup()
                 }
 
                 this.cleanupScheduled = true;
-                Task.Run(async () => await this.CollectExpiredEntriesAsync().ConfigureAwait(false));
+                _ = this.CollectExpiredEntriesAsync(token);
```

Comment:
This will start a task that will do the collection without waiting for the cleanup to happen -- is this the design?

Reply:
Yes see previous implementation did a fire and forget Task.Run